modnet/models.py

```diff
@@ -46,6 +47,9 @@ def __init__(
             targets: A nested list of targets names that defines the hierarchy
                 of the output layers.
             weights: The relative loss weights to apply for each target.
+            task_type: Dictionary defining the target types (classification or regression).
+                Should be constructed as follows: key: string giving the target name; value: integer n,
+                 with n=0 for regression and n>=2 for classification with n the number of classes.
```

Comment:
I like this more than corresponding `MODData` version, but perhaps a different name is better? `num_classes`?